htdocs/lib2/menu.class.php

```diff
@@ -34,7 +36,7 @@ public function __construct()
         // add country-dependent town list for small map
         $country = $login->getUserCountry();
         if ($opt['map']['towns']['enable'] &&
-            isset($build_map_towns_menu) && $build_map_towns_menu &&   // optimization
+            isset($build_map_towns_menu) && $build_map_towns_menu && // optimization
```

Comment:
this is because of the "1 space after logical operator" rule. (Squiz.WhiteSpace.LogicalOperatorSpacing.TooMuchSpaceAfter)
If this is really a deal breaker we have to write our own sniff or solve it manually